models/intermediate/int_zendesk__ticket_schedules.sql

```diff
@@ -15,43 +15,41 @@ with ticket as (
   select *
   from {{ ref('stg_zendesk__schedule') }}
 
-
-), default_schedule_events as (
+), default_schedules as (
 -- Goal: understand the working schedules applied to tickets, so that we can then determine the applicable business hours/schedule.
 -- Your default schedule is used for all tickets, unless you set up a trigger to apply a specific schedule to specific tickets.
 
 -- This portion of the query creates ticket_schedules for these "default" schedules, as the ticket_schedule table only includes
 -- trigger schedules
+  select 
+    schedule_id,
+    source_relation
+  from (
+    
+    select
+      schedule_id,
+      source_relation,
+      row_number() over (partition by source_relation order by created_at) = 1 as is_default_schedule
+    from schedule
 
-{% if execute %}
-
-    {% set default_schedule_id_query %}
-        with set_default_schedule_flag as (
-          select 
-            row_number() over (order by created_at) = 1 as is_default_schedule,
-            id
-          from {{ source('zendesk','schedule') }}
-        )
-        select 
-          id
-        from set_default_schedule_flag
-        where is_default_schedule
-
-    {% endset %}
-
-    {% set default_schedule_id = run_query(default_schedule_id_query).columns[0][0]|string %}
+  ) as order_schedules
+  where is_default_schedule
```

Comment:
I noticed the results of this query change between your changes and prod. I understand the need to remove the run query as more than one default id may be returned and it ensure we don't need to look at the source, but the results do seem to differ for the `zendesk_yashwanth` schema.

![image](https://github.com/fivetran/dbt_zendesk/assets/74217849/b15b94b3-73a8-47c5-9135-3d963d8e0f35)

Reply:
Ohhh so schedule `360000389531` is deleted... 

<img width="768" alt="image" src="https://github.com/fivetran/dbt_zendesk/assets/65564846/3ff39c58-dc03-4650-8fd8-1210ebdbf3c7">

Because the query is now pointing to the staging model, it's [filtering](https://github.com/fivetran/dbt_zendesk_source/blob/main/models/stg_zendesk__schedule.sql#L41) out deleted schedules and therefore selecting `11630270684308`

So the question is, should we stick to the pre-existing behavior, which will select the very first (but potentially deleted) schedule as the default, or is this kinda a bug?